app/views/achievements/_grid.html.haml

```diff
@@ -1,11 +1,13 @@
 - player ||= nil
 .row.achievement_grid
   - achievements.each do |achievement|
-    .col-md-3.achievement
+    .col-md-4.achievement
       .panel.panel-default{class: achievement_class(achievement, player)}
-        .panel-heading=achievement.title
+        .panel-heading
+          = achievement.title
+          .points.pull-right{title: t("achievement_points")}
```

Comment:
If you're interested why, it's because symbols are immutable, so the system can store them in the same place in memory. Three instances of `:symbol` just point to the same place. Three instances of `"string"` must be stored in separate places and use three times the memory. You can imagine that keys in particular (which could easily be repeated many times throughout a project) are preferred as symbols. Unlikely to make a difference on a project of this scale, but that's the convention.